lib/gooddata/lcm/actions/synchronize_cas.rb

```diff
@@ -17,11 +17,22 @@ class SynchronizeComputedAttributes < BaseAction
 
         description 'Synchronization Info'
         param :synchronize, array_of(instance_of(Type::SynchronizationInfoType)), required: true, generated: true
+
+        description 'Specifies whether to transfer computed attributes'
+        param :include_computed_attributes, instance_of(Type::BooleanType), required: false, default: true
       end
 
       class << self
         def call(params)
+          # set default value for include_computed_attributes
+          # (we won't have to do this after TMA-690)
+          include_ca = params.include_computed_attributes
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)